sections/faqs/reverse-selectors.js

```diff
@@ -51,6 +51,25 @@ const ReverseSelectors = () => md`
 
   We could have nested the color-changing rule within our Link component, but then we'd have to
   consider both sets of rules to understand why Icon behaves as it does.
-`
 
-export default ReverseSelectors
+  ### Caveat
+
+  This behaviour is only supported within the context of *Styled* Components: attempting to mount 
+  `B` in the following example will fail (`Cannot call a class as a function`) because the referenced
+  component is an instance of React.Component not a Styled Component
+
+  \`\`\`react
```

Comment:
We probably don't want this to be a live-editable example, since it won't work :smile:

Reply:
TIL this is the fence for a live example!